app/facades/item_facade.rb

```diff
@@ -0,0 +1,29 @@
+class ItemFacade
+
+  def self.all_items
+    all_items = items_data[:data].map do |data|
+      ItemPoro.new(data)
+    end
+  end
+
+  def self.find_items(merchant_id)
```

Comment:
This is a little nit picky but it might be a little clearer to name this method something like `find_merchants_items` so that when it's called in the controller, we know exactly what it's supposed to be looking for/returning.